### PR TITLE
Create classes and set up associations

### PR DESCRIPTION
**Pull request by lenkon**
- Create a class Classroom with a @label instance variable which is initialized in the constructor.
- Create class associations between Classroom and Student.
- Create a class Book with @title and @author instance variables which are initialized in the constructor.
- Create a class Rental with @date instance variable which is initialized in the constructor.
- Create class associations between Person and Book.

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
```
Comment (TadesseDev) [inline rental.rb]: - [ ] Good job adding the getter and setter for the date instance variable. Please also set the Getters and Setters for `:book, and :person` instance variables.  This creates a lot of flexibility in how rental is created and accessed.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @lenkon 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x]  Correct GitFlow  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] Please try to separate your code into different levels of commits. Remember your commits are going to help you look back and debug your code. It's a good idea to commit after every visible change.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: rental.rb (hunk 1 above)
Comment (lenkon) [reply rental.rb]: Hi @TadesseDev ,

Thanks for your feedback and optional suggestion. I have updated the instance variables getter and setter.

Best Regards,
K. Inoulen Shanongba

Comment (TadesseDev) [reply rental.rb]: Good job @lenkon, As I have mentioned in the optional changes, Please make sure to have more commits in your project. Try to commit once you accomplish something and move on to the next one. this is a good practice and that is why `git` exists anyway.

**Review by julie-ify — APPROVED** [review]
Hi @lenkon,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**